helm/chaos-mesh/templates/chaos-daemon-rbac.yaml

```diff
@@ -58,16 +58,45 @@ metadata:
     {{- include "chaos-mesh.labels" . | nindent 4 }}
     app.kubernetes.io/component: chaos-daemon
 spec:
-  privileged: false
-  allowPrivilegeEscalation: true
+  {{- if .Values.chaosDaemon.privileged }}
+  allowedCapabilities:
+    - SYS_PTRACE
+  {{- else }}
   allowedCapabilities:
-    - '*'
+    - SYS_PTRACE
+    - NET_ADMIN
+    - MKNOD
+    - SYS_CHROOT
+    - SYS_ADMIN
+    - KILL
+    # CAP_IPC_LOCK is used to lock memory
+    - IPC_LOCK
+  {{- end }}
+  allowedHostPaths:
+    - pathPrefix: {{ .Values.chaosDaemon.socketPath | default "/var/run/docker.sock" }}
+      readOnly: false
+    - pathPrefix: /sys
+      readOnly: false
+    {{- if .Values.bpfki.create }}
+    - pathPrefix: /etc/localtime
+      readOnly: false
+    - pathPrefix: /lib/modules
+      readOnly: false
+    - pathPrefix: /usr/src
+      readOnly: false
+    {{- end }}
+  allowPrivilegeEscalation: true
+  {{- if .Values.chaosDaemon.hostNetwork }}
   hostNetwork: true
+  {{- end }}
+  {{- if .Values.bpfki.create }}
   hostPorts:
-  - max: 65535
-    min: 0
+  - max: {{ .Values.bpfki.grpcPort }}
+    min: {{ .Values.bpfki.grpcPort }}
```

Comment:
Are both max and min set to ".Values.bpfki.grpcPort"?  In this case, is "max" equal to "min"?

Reply:
Yes. However this is the correct way to specify a single allowed host port.

Reply:
This config should contain the ports of chaos-daemon.  https://github.com/chaos-mesh/chaos-mesh/blob/master/helm/chaos-mesh/values.yaml#L88-L89